### ISSUE TITLE
How Do Agricultural Production Processes Affect Carbon Emissions?

### ISSUE THREAD

**Issue by tutututuni** [JGCRI/gcam-core#453]
In the GCAM Model, I Found That Modifying Agproductchange.xml Can Adjust the Effects of Irrigation or Fertilization on Yield Changes Independently. However, the fertilization process generates substantial agricultural carbon emissions. How does the model quantify the proportion of yield increases attributable to fertilization? Additionally, how is the carbon emission from fertilizers quantitatively assessed, and where is the application rate of nitrogen fertilizers specified? We are particularly interested in how the emission module calculates nitrogen fertilizer emissions. I aim to extend the agricultural module of the GCAM model.
     In summary, I seek to understand how the GCAM model quantifies the impact of fertilization on carbon emissions, given that the model outputs carbon emissions for nitrogen.
![gca](https://github.com/user-attachments/assets/0f1dacd0-f87a-451b-ab0d-473191ea341f)

**Comment by realxinzhao** [comment]
N2O emissions are calculated per ag technology, with initially "observed/derived" emission factors (e.g., t N2O per t of corn production) applied to production.

**Comment by Tonyzhao610** [comment]
Thank you very much！！
So the CO2 emissions of crops are calculated based on yields, regardless of whether the change in yield is affected by irrigation or fertilization?

**Comment by pkyle** [comment]
I just want to note in this thread, we do have an open Postdoc position for those interested in improving the nitrogen tracking in GCAM: https://umces.peopleadmin.com/postings/1781
In response to the specific questions, I think first we need to be clear about what Is meant by carbon emissions. Both I and @realxinzhao assume that we are referring to GHG emissions, i.e. any N2O emissions are multiplied by a factor of about 300 to get the CO2 equivalent. Right now in GCAM, the only GHG emissions directly produced by crop production are from non-CO2 GHGs: N2O for all crops, CH4 mostly for paddy rice. The model keeps track of CO2 from fertilizer production in the ammonia sector, from agricultural energy use in the agricultural energy use sector, and from land use change in the land/carbon cycle model.
To the original question, the model’s representation of each agricultural production technology is pretty simple: for each unit of crop production (e.g. per each 1 kg of corn produced), a set of inputs is required, including land, fertilizer, and water. These inputs are non substitutable (i.e. a Leonteif production function) and as a default are assumed linearly related. If `AgProdChange` is increased, all else equal, this will increase the yield (i.e. reduce the land input), and all other inputs and emissions are indicated per kilogram of crop produced, so they will not change per unit crop produced, but per unit land area will scale linearly with the assumed yield change.
